.travis.yml

```diff
@@ -10,7 +10,7 @@ python:
   - "3.8"
 
 env:
-  - DJANGO_SETTINGS_MODULE="oilandrope.settings"
+  - SECRET_KEY="de68z30c(3nbj*k4=lumea8hztcy_6%d0epx^w$jc&s)wygezo" DJANGO_SETTINGS_MODULE="oilandrope.settings"
```

Comment:
Shouldn't this key be more private?

Reply:
It's only for testing, Django needs a Secret_key in order to work